src/i18n/sv.js

```diff
@@ -0,0 +1,102 @@
+export default{
+	error : {
+		forgotPassword : {
+			"too_many_requests" : "Du har nått gränsen för maximalt antal försök att ändra ditt lösenord. Var vänlig vänta innan du försöker igen.",
+			"lock.fallback" : "Något gick fel när vi försökte ändra ditt lösenord."
+		},
+		login : {
+			"blocked_user" : "Användaren är spärrad.",
+			"invalid_user_password" : "Inloggningsuppgifterna är felaktiga.",
+			"lock.fallback" : "Något gick fel vid inloggningsförsöket.",
+			"lock.invalid_code" : "Fel kod.",
+			"lock.invalid_email_password" : "Fel e-postadress eller lösenord.",
+			"lock.invalid_username_password" : "Fel användarnamn eller lösenord.",
+			"lock.network" : "Kunde inte kontakta servern. Var vänlig kontrollera din uppkoppling och försök igen.",
+			"lock.popup_closed" : "Popup-fönster stängt. Försök igen.",
+			"lock.unauthorized" : "Åtkomst nekad. Försök igen.",
+			"password_change_required" : "Du behöver ändra ditt lösenord då detta är första gången du loggar in eller för att ditt lösenord har gått ut.", // TODO: verify error code
+			"password_leaked" : "Din konto har blivit låst då ditt lösenord har lagts upp på en annan webbsida. Du har fått e-post med instruktioner om hur du låser upp ditt konto.",
+			"too_many_attempts" : "Ditt konto har blivit låst efter att flera felaktiga inloggningsförsök har gjorts."
```

Comment:
At row 8 the word "blocked" is translated to "spärrad".  I think that is better than "låst", but at least use the same translation.  And, I would use a more terse passive form for the latter part: "Ditt konto har blivit spärrat efter flera felaktiga inloggningsförsök."